styles.css

```diff
@@ -304,6 +334,159 @@ h2 {
   color: #4472e7;
 }
 
+#contact-btn:hover {
+  color: white;
+  background-color: #4472e7;
+  border: 1px solid #4472e7;
+}
+
 .contact-message {
   height: 100px;
 }
+
+@media (min-width: 420px) {
+  .intro-section {
+    background-image: url("images/main-desktop-background.png");
+    height: 80vh;
+  }
+
+  .menu1 {
+    visibility: visible;
+    display: flex;
+  }
+
+  .menu1 ul {
+    display: flex;
+  }
+
+  .menu1 li {
+    list-style: none;
+    padding: 10px;
+    text-decoration: none;
+  }
+
+  .menu1 li a {
+    text-decoration: none;
+    color: #344563;
+    font-size: 1.2em;
+  }
+
+  .menu1 li a:hover {
+    color: #4472e7;
+  }
+
+  .menu {
+    display: none;
+  }
+
+  .contact-section {
+    background-image: url("images/contact-desktop-background.png");
+    background-size: cover;
+  }
+}
+
+@media (min-width: 750px) {
+  .languages-box2 {
+    padding-top: 40px;
+    width: 50%;
+  }
```

Comment:
- [ ] I'd suggest that you ensure that the container of this section has a width of about 60%. The other section/block should cover around 40% of the width of the page.